packages/design-system-docs/src/pages/components/VerticalNav/_VerticalNav.docs.scss

```diff
@@ -58,8 +58,10 @@ if you're using the React components.
 - If your navigation list is longer than 3 items, consider using a skip navigation link. This allows screen reader and keyboard users to skip to the main content area(s).
 - If a skip navigation link is not an option, consider using:
   * A valid, descriptive page header
-  * Landmark regions like `<header>`, `<nav>`, `<main>`, `<aside>`, `<footer>`
-  * A `role` attribute like `<div role="navigation">`
+  * Landmark regions like `<header>`, `<main>`, `<aside>`, `<footer>`
+- When naming the optional `ariaNavLabel` prop, keep in mind:
+    * This prop is necessary only when there is more than one nav element on a page.
```

Comment:
Thinking out loud here...

I'm curious about the usage of this `aria-label`. This feels like something that will almost certainly never be used even with this information in the documentation. Is there a way to add more emphasis here so it stands out more? Or maybe have a default `aria-label` text that can be changed or turned off?

Reply:
Good callout, @zarahzachz. I struggled with this a bit too when thinking about it. I don't know how many of the applications/sites that use this component have another menu on the page.

I would be fine going the route of having it there by default because even if you only have one menu, having it named isn't a bad thing. It only provides more context for people who use screen readers.

Where I wondered if providing a default might be difficult is: What do we name it? We don't have any context of what will be in the navigation. We could call it something like "Main" or "Primary" and hope that people update it to fit their needs.

Reply:
Per a Slack conversation, it seems like it is better to have a region with no name than a region that is mis-named. So we should not provide a default for aria-label, for now.